src/main/java/com/authlete/common/dto/Service.java

```diff
@@ -815,6 +817,52 @@ public Service setRegistrationEndpoint(URI endpoint)
     }
 
 
+    /**
+     * Get the URI of the registration management endpoint. If dynamic
+     * client registration is supported, and this is set, this URI will be 
+     * used as the basis of the client's management endpoint by appending 
+     * {@code /clientid/} to it as a path element. If this is unset,
+     * the value of {@code registrationEndpoint} will be used as the
+     * URI base instead. 
+     *
+     * @return
+     *         The base URI of the registration management endpoint.
+     *         
+     * @since
+     *         2.XX
```

Comment:
- No line break is needed between `@since` and `2.XX`.
- Update `2.XX` to `2.39`.

Reply:
Since we can't predict exactly the target version where a feature set will be added, I leave the versions off of functions so that they can be updated along with the documentation when the request is merged. This way it's more obvious that there's a value that needs updating here, whereas as value of "2.39" could be missed if this actually goes out in "2.40".